hudi-hadoop-common/src/main/java/org/apache/hudi/io/storage/HoodieParquetConfig.java

```diff
@@ -55,8 +49,4 @@ public HoodieParquetConfig(T writeSupport, CompressionCodecName compressionCodec
     this.compressionRatio = compressionRatio;
     this.dictionaryEnabled = dictionaryEnabled;
   }
-
-  public boolean dictionaryEnabled() {
```

Comment:
Removing this and using @Getter will change the method signature from `dictionaryEnabled` to `isDictionaryEnabled`.